litellm_proxy/README.md

```diff
@@ -0,0 +1,20 @@
+# LiteLLM Proxy Docker
+
+All of our embeddings and LLM calls will now pass through this proxy. See [confluence page](https://idinsight.atlassian.net/wiki/spaces/PD/pages/2452488357/LiteLLM+Proxy+Server) for details.
+
+## How to run
+
+1.  Pull LiteLLM proxy image
+
+        docker pull ghcr.io/berriai/litellm:main-v1.34.6
```

Comment:
Will you add this as another service on docker-compose?

Reply:
yup